<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactManager</class>
 <widget class="QMainWindow" name="ContactManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="placeholderText">
       <string>Enter contact name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numberLineEdit">
      <property name="placeholderText">
       <string>Enter contact number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="contactTableView"/>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add Contact</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editButton">
      <property name="text">
       <string>Edit Contact</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeButton">
      <property name="text">
       <string>Remove Contact</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
